reporting/sending_metrics_report.md

```diff
@@ -0,0 +1,19 @@
+# Sending Metrics Report
+
+## Configurations
+
+Sending Metrics Report is an automated task in UgandaEMR, however it requires setting some facility Credentials ie. Analytics Username and Password and DHIS2 Facility UUID
+
+To set the Analytics User name and password,
+
+1. Got to Legacy System Administration-&gt;Mantainance-&gt;Settings-&gt;UgandaemrSync.
+
+2. Set the Analytics Username and Password as METS and METS1234 respectively and save.
+
+To set the DHIS2 Facility UUID,
```

Comment:
@dbaluku This should be step 3, set the Facility DHIS2 uuid rather than a new section on its own

Reply:
Done